utils.ts

```diff
@@ -38,6 +46,76 @@ export const makeQueryPeliasCompatible = (queryString: string): string => {
   return queryString.replace(/@/g, ' ').replace(/&/g, ' ')
 }
 
+export const hereResultTypeToPeliasLayer = (resultType: string): string => {
```

Comment:
Could this also not be moved to the geocoder package?

Reply:
How did this even get here? It's not even being used.